Frameworks/UIKit/UIViewController.mm

```diff
@@ -715,7 +715,7 @@ - (NSString*)nibName {
         NSString* ret = nil;
         NSString* nibPath;
 
-        char* ourClass = IwStrDup(object_getClassName(self));
+        char* ourClass = _strdup(object_getClassName(self));
```

Comment:
agreed.

---
In reply to: [113573646](https://github.com/Microsoft/WinObjC/pull/2636#discussion_r113573646) [](ancestors = 113573646)